qa/src/test/java/org/eclipse/kapua/service/user/steps/TestCredentials.java

```diff
@@ -35,4 +48,54 @@ public String getPassword() {
     public void setPassword(String password) {
         this.password = password;
     }
+
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    public CredentialStatus getStatus() {
+        if (isEnabled()) {
+            return CredentialStatus.ENABLED;
+        } else {
+            return CredentialStatus.DISABLED;
+        }
+    }
+
+    public Date getExpirationDate() {
```

Comment:
Much ceaner I agree.
I have changed it, it will be in next PR with more tests.
TNX, good code review